rest-e2e-tests/streamr-api-clients.js

```diff
@@ -235,7 +235,16 @@ class Streams {
             .withBody(body)
     }
 
-    makePublic(id) {
+    grantSubscribe(id) {
```

Comment:
The method `makePublic` grants *public* permissions (=to the anonymous user), so the names `grantSubscribe` and `grantGet` don't describe what the method does very well. If you want to split the `makePublic` into atomic components, then you should either:

- Use a more accurate name like `grantPublicSubscribe` and `grantPublicGet`, or 
- Refactor them to `grantSubscribe(id, user)`, where setting `user` to `null` would grant the anonymous permission

Personally I might just keep `makePublic`... :)